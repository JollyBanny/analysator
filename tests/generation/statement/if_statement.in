var a: integer = 1; b: integer = 10; c: integer = 20;

begin
    if(a > b) then
        writeln('a bigger than b')
    else
        writeln('b bigger than a');

    if(c > b) then
        writeln('c bigger than b')
    else
        writeln('b bigger than c');
end.