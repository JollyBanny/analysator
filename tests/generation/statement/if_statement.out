b bigger than a
c bigger than b
